ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -235,7 +235,8 @@ function prepare_next_steps() {
       -t "${SHARED_DIR}" \
       "${dir}/auth/kubeconfig" \
       "${dir}/auth/kubeadmin-password" \
-      "${dir}/metadata.json"
+      "${dir}/metadata.json" \
+      "${dir}/.openshift_install.log"
```

Comment:
`.openshift_install.log` has a bigger size than other files, while `SHAORED_DIR` is not supposed to save some big size of files. To reduce impact as much as possible, suggest to some condition check, e.g: only vsphere + template is set in install-config.ymal, we will copy it.